lib/core/utils.js

```diff
@@ -109,6 +109,17 @@ RESUtils.isMatchURL = function(moduleID) {
 
 	return included;
 };
+RESUtils.minifyOptions = function(obj) {
+	var min = {};
+	if (obj) {
+		for (var key in obj) {
+			if ('value' in obj[key]) {
+				min[key] = {value: obj[key].value};
+			}
+		}
+	}
+	return min;
+};
```

Comment:
hmm, I'm preeeettty sure this is safe, but I'm trying to think of a scenario where we might need to save anything more complex beyond just 'value'.... I don't think there is, and we could always make value a complex object if need be... but @andytuba @mc10 or anyone else, any thoughts on that before we merge?